common/source/docs/common-serial-led-ProfiLED.rst

```diff
@@ -0,0 +1,12 @@
+.. _common-serial-led-ProfiLED:
+
+ProfiLED
+========
+
```

Comment:
this feature available in Copter 4.0 and later

Reply:
Done